### ISSUE TITLE
[Bugs] Feed items truncated when Length of entries is 1

### ISSUE THREAD

**Issue by mystralkk** [Geeklog-Core/geeklog#574]
**Reported by jrivett on 29 Feb 2012 16:03**

**Version:** 1.8.1

[Mantis Bugtracker #1431](http://project.geeklog.net/tracking/view.php?id=1431)

**Comment by mystralkk** [comment]
**Comment posted by jrivett on 29 Feb 2012 16:10**

My first attempt to post this bug (#1430) resulted in an error indicating that my attachment (lib-syndication.php) was not allowed.  The message instructed me to press the Back button and change it.  I did so, and tried to remove the attachment, but was unable to do so.  This is apparently a bug in the bugtracker software.  I then renamed the file with a TXT extension and selected that as the attachment, which worked.  However, this created a duplicate bug: this one, #1431.  That looks like yet another bug to me.  Just sayin&#039;.  Anyway, sorry for the duplicate but I don&#039;t think it was my fault.  I found no way to delete #1430, so that&#039;s something for the bugtracker admins to deal with I guess.
